HISTORY.md

```diff
@@ -37,6 +37,9 @@
  * Introduce Policy Methods for MOEA/D-DE
    ([#293](https://github.com/mlpack/ensmallen/pull/293)).
 
+ * ZDT4 Evaluate Fix
+   ([#298](https://github.com/mlpack/ensmallen/pull/298)).
```

Comment:
ZDT wasn't part of a release, so no need to include the PR in the HISTORY.

Reply:
@zoq I think he's trying to avoid running into having the CI pick up an error if HISTORY wasn't modified.

Reply:
Both of you are correct. In either case, this change isn't anything significant, so we can ignore HISTORY.

Reply:
Right, happy to ignore the failing build in this case, in favor of a cleaner HISTORY.